src/Core/Migration/V6_4/Migration1665267882RenameCountryVat.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Shopware\Core\Migration\V6_4;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Migration\MigrationStep;
+use Shopware\Core\Migration\Traits\ImportTranslationsTrait;
+use Shopware\Core\Migration\Traits\Translations;
+
+class Migration1665267882RenameCountryVat extends MigrationStep
+{
+    use ImportTranslationsTrait;
+
+    public function getCreationTimestamp() : int
+    {
+        return 1665267882;
+    }
+
+    public function update(Connection $connection) : void
+    {
+        $results = $connection->fetchFirstColumn('SELECT id FROM country WHERE iso = :iso AND iso3 = :iso3', ['iso' => 'VA', 'iso3' => 'VAT']);
+
+        $countryId = $results[0];
+
+        $translations = new Translations(
+            [
+                'country_id' => $countryId,
+                'name'       => 'Staat Vatikanstadt',
+            ], [
+                'country_id' => $countryId,
+                'name'       => 'Vatican City',
+            ]
+        );
+        $this->importTranslation('country_translation', $translations, $connection);
```

Comment:
With this migration you would also overwrite all manual edits that may be done to the countries name already, please add a check and only change the name if it currently is the rather bad default name, and also add a testcase that manual edits of the country name are not overwritten in your migration

Reply:
I have adjusted the migration so that only the translations with the default value and existing languages are updated.
The test is also reworked accordingly.